<launch>
   <node name="gaze_fixing_node" pkg="manipulation_tutorials" type="gazed_point_fixing_node" respawn="false" output="screen"/>
   <node name="gazing_action" pkg="gaze_service" type="gaze_action_server" respawn="false" />
</launch>
